android/src/main/java/vn/luongvo/kmm/survey/android/ui/theme/AppDimensions.kt

```diff
@@ -5,6 +5,7 @@ import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 
 class AppDimensions {
+    val paddingSmallest: Dp = 8.dp
```

Comment:
@luongvo Should we use `paddingTiny` ? 🤔

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/68/commits/6af9f7ebe8f4beda0525c54f22e00de777a8d9ce